lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/dao/DBOTeamDAOImpl.java

```diff
@@ -355,6 +364,29 @@ public long getCount() throws DatastoreException {
 		return dtos;
 	}
 
+	@Override
+	public List<String> getIdsForMember(String principalId, long limit, long offset, TeamSortOrder sortBy, Boolean ascending) {
+		MapSqlParameterSource params = new MapSqlParameterSource();
+		params.addValue(COL_GROUP_MEMBERS_MEMBER_ID, principalId);
+		String query = buildGetQuery(SELECT_IDS_FOR_MEMBER_PAGINATED, limit, offset, sortBy, ascending);
+		return namedJdbcTemplate.queryForList(query, params, String.class);
+	}
+
+	protected static String buildGetQuery(String query, long limit, long offset, TeamSortOrder sortBy, Boolean ascending) {
```

Comment:
Please consider writing two static strings (select with sort, select without sort) rather than building the string at run time.